api-specificatie/openapi.yaml

```diff
@@ -1195,7 +1195,7 @@ components:
     AdresseerbaarObject_links:
       type: object
       properties:
-        hoofdAdres:
+        adressen:
```

Comment:
Wat moet adressen bevatten: alleen hoofd adres, alleen nevenadressen of alles? Hoe weet een gebruiker dan wat wat is?

Reply:
Klopt Mark. Dit dilemma heb ik ook in #42 beschreven en staat op mijn lijstje om vanmiddag te bespreken.